jip/util.py

```diff
@@ -71,6 +71,7 @@ def download_string(url):
     import requests
     try:
         response = requests.get(url, headers={ 'User-Agent': JIP_USER_AGENT})
+        response.encoding = 'utf-8'
```

Comment:
Ok. But it doesn't seem to matter what 'requests' assigns. I've tried the code without setting the encoding attribute and it works. I think it is just metadata about the response that you can query if you want. But it doesn't affect how the data is interpreted.